components/clp-package-utils/clp_package_utils/general.py

```
@@ -584,6 +585,16 @@ def validate_webui_config(
     validate_port(f"{WEBUI_COMPONENT_NAME}.port", clp_config.webui.host, clp_config.webui.port)
 
 
+def validate_mcp_server_config(clp_config: CLPConfig, logs_dir: pathlib.Path):
+    if not logs_dir.exists():
+        error_msg = f"{MCP_SERVER_COMPONENT_NAME} logs directory at {str(logs_dir)} is missing."
```

Ruff (0.14.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)